Dockerfile-manager

```diff
@@ -5,6 +5,7 @@ RUN yum -y update && \
     yum -y install rh-python36 postgresql && \
     rm -rf /var/cache/yum/*
 
+ENV APP_VERSION=latest
```

Comment:
there is already very similar `ENV INSIGHTS_VULNERABILITY_VERSION=latest` 2 lines below, is this ok?

Reply:
No, it's mistake, fixed.